src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/EventRoutes.scala

```diff
@@ -0,0 +1,91 @@
+package ch.epfl.bluebrain.nexus.kg.routes
+
+import java.util.UUID
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.http.javadsl.server.Rejections.validationRejection
+import akka.http.scaladsl.marshalling.sse.EventStreamMarshalling._
+import akka.http.scaladsl.model.headers.`Last-Event-ID`
+import akka.http.scaladsl.model.sse.ServerSentEvent
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Directive1, Route}
+import akka.persistence.query._
+import akka.persistence.query.scaladsl.EventsByTagQuery
+import akka.stream.scaladsl.Source
+import ch.epfl.bluebrain.nexus.admin.client.types.Project
+import ch.epfl.bluebrain.nexus.iam.client.config.IamClientConfig
+import ch.epfl.bluebrain.nexus.iam.client.types.{AccessControlLists, Caller, Permission}
+import ch.epfl.bluebrain.nexus.kg.config.AppConfig
+import ch.epfl.bluebrain.nexus.kg.directives.AuthDirectives._
+import ch.epfl.bluebrain.nexus.kg.resources.Event
+import ch.epfl.bluebrain.nexus.kg.resources.Event.JsonLd._
+import io.circe.syntax._
+import io.circe.{Encoder, Printer}
+
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+class EventRoutes(acls: AccessControlLists, caller: Caller)(implicit as: ActorSystem,
+                                                            project: Project,
+                                                            config: AppConfig) {
+
+  private val printer: Printer = Printer.noSpaces.copy(dropNullValues = true)
+
+  private val pq: EventsByTagQuery =
+    PersistenceQuery(as).readJournalFor[EventsByTagQuery](config.persistence.queryJournalPlugin)
+
+  private val resourcesRead: Set[Permission]    = Set(Permission.unsafe("resources/read"))
```

Comment:
can we call it just `private val read: Set[Permission]`, just to have the same as in CommonRoutes